KarhooUISDK/Screens/BookingScreen/QuoteListMVP/KarhooQuoteListPresenter.swift

```diff
@@ -96,6 +105,11 @@ final class KarhooQuoteListPresenter: QuoteListPresenter {
             let sortedQuotes = quoteSorter.sortQuotes(quotesToShow, by: selectedQuoteOrder)
             quoteListView?.showQuotes(sortedQuotes, animated: animated)
         }
+        
+        if fetchedQuotes.status == .completed {
```

Comment:
This function is getting long now. is it possible to create a function e.g. `handleVehiclePolling` and move this if statement there? It would help to understand the logic in `updateViewQuotes`

Reply:
Sorted